embedded-io/Cargo.toml

```diff
@@ -14,6 +14,7 @@ categories = [
 [features]
 std = ["alloc"]
 alloc = []
+defmt-03 = ["dep:defmt-03"]
```

Comment:
IIUC this is equivalent to the old code. Why is it better?

Reply:
I believe this is better because it makes `defmt-03` feature declaration explicit for anyone looking at cargo.toml -- one can see all features at a glance.  It also means that if additional dependencies are needed for the `defmt-03` feature, they can be simply listed at the end of the list.  In short, I prefer this form simply to follow Rust-way of "no implicit surprises" :)

Reply:
A bit redundant but I sympathize with the case of seeing all features at a glance.